bindgen/spec.yml

```
@@ -65,6 +65,7 @@ typeAliases:
   AppCredentialsToken: std::string
   Email: std::string
   SchemaVersion: int64_t # is uint64_t in C++ but -1 is used as a sentinel.
+  PortType: uint64_t # is uint_fast16_t in C++ but we don't want to add more primitive types and node.js will not parse URLs with port number > 65535
```

This spec is not specific to Realm JS, so I think this comment shouldn't mention Node.js.
```suggestion
  PortType: uint64_t # is uint_fast16_t in C++ but we don't want to add more primitive types and expect SDKs to reject port number > 65535
```